File: 2021/November/Day 28/palindrome.cs
# source : https://leetcode.com/problems/palindrome-number/submissions/
public class Solution {
    public bool IsPalindrome(int x) {
        String original = x.ToString();
        String s = "";
        if(x==0){return true;}
        while(x>0)
        {
            var v = x%10;
            s = s+ (v.ToString());
            x = x/10;
        }
        return original ==s;
    }
}

File: 2022/March/Day 8/palindrome.cpp
class Solution {
public:
    bool isPalindrome(int x) {

        string s = "";
        while(x)
        {
            s+=to_string(x%10);
            x/=10;
        }
        cout<<s;
        for(int i=0;i<size(s)/2;i++)
        {
            if(s[i]!=s[size(s)-1-i])
            {
                return false;
            }
        }
        return true;

    }
};
